ui/components/multichain/multichain-import-token-link/multichain-import-token-link.js

```diff
@@ -0,0 +1,85 @@
+import React, { useContext } from 'react';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from '../../ui/box/box';
+import { ButtonLink, ICON_NAMES } from '../../component-library';
+import {
+  AlignItems,
+  DISPLAY,
+  Size,
+} from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { IMPORT_TOKEN_ROUTE } from '../../../helpers/constants/routes';
+import { detectNewTokens } from '../../../store/actions';
+import { MetaMetricsContext } from '../../../contexts/metametrics';
+import { EVENT, EVENT_NAMES } from '../../../../shared/constants/metametrics';
+import {
+  getIsTokenDetectionSupported,
+  getIsTokenDetectionInactiveOnMainnet,
+} from '../../../selectors';
+
+export const MultichainImportTokenLink = ({ className, ...props }) => {
+  const trackEvent = useContext(MetaMetricsContext);
+  const t = useI18nContext();
+  const history = useHistory();
+
+  const isTokenDetectionSupported = useSelector(getIsTokenDetectionSupported);
+  const isTokenDetectionInactiveOnMainnet = useSelector(
+    getIsTokenDetectionInactiveOnMainnet,
+  );
+
+  const isTokenDetectionAvailable =
+    isTokenDetectionSupported ||
+    isTokenDetectionInactiveOnMainnet ||
+    Boolean(process.env.IN_TEST);
+  return (
+    <Box
+      className={classnames('multichain-import-token-link', className)}
+      {...props}
+    >
+      <Box display={DISPLAY.FLEX} alignItems={AlignItems.center}>
+        <ButtonLink
+          size={Size.MD}
+          startIconName={ICON_NAMES.ADD}
+          onClick={() => {
+            history.push(IMPORT_TOKEN_ROUTE);
+            trackEvent({
+              event: EVENT_NAMES.TOKEN_IMPORT_BUTTON_CLICKED,
+              category: EVENT.CATEGORIES.NAVIGATION,
+              properties: {
+                location: 'Home',
+              },
+            });
+          }}
+        >
+          {isTokenDetectionAvailable
+            ? t('importTokens')
+            : t('importTokens').charAt(0).toUpperCase() +
```

Comment:
Instead of slicing up the string, can we use the `"importTokensCamelCase"`?  Why is this string reliant upon `isTokenDetectionAvailable`?